test/tests.js

```diff
@@ -10,92 +10,109 @@ const defaultPluginOptions = {
   }
 };
 
-function transform(code, transformOptions = {}, pluginOptions = defaultPluginOptions) {
-  return babelTransform(code, {
+async function transform(code, transformOptions = {}, pluginOptions = defaultPluginOptions) {
```

Comment:
i'm assuming you've tried it where it's used in the frontend alraedy, and it deals with this new promise via async/await as well...or `.then()...catch()`?

Reply:
ya front end worked! for some reason i could only get the tests to run if I used async